common/v2/components/WalletList/WalletList.tsx

```diff
@@ -39,7 +39,7 @@ const Description = styled.p`
 const WalletsContainer = styled.div`
   display: flex;
   justify-content: center;
-  flex-wrap: wrap;
+  flex-wrap: wrap-reverse;
```

Comment:
According to the card and Luker the wallets should be in the following rows:

MM, WC
Ledger, Trezor
Keystore, Private Key, mnem
View only

Reply:
Isn’t it the case ?

Reply:
No, it should be 2-2-3-1, you have 2-3-3 😄